### ISSUE TITLE
change request: start of outline and brim

### ISSUE THREAD

**Issue by fluxline** [daid/LegacyCura#885]
the start of the print with an outline or brim starts at the inside of the print causing the G1 F200 E3 in start code that is put down to interfere with the real print. in some cases it is caught, and the outline is pulled up.  would it be possible to start on the outside, starting from the point closest to home position. obviously, not critical.

**Comment by daid** [comment]
Brim is printed outside in, except when the "bed center is 0,0" as far as I know.

**Comment by Dim3nsioneer** [comment]
Brims on the inside of bigger holes are also printed outside in which is the wrong direction.

**Comment by fluxline** [comment]
i agree, the brim is printed from the outside in, it is more where it first starts.  the start is often at an inside position, or at the farthest point from home and the initial nozzle clearing sits at some inside part of the remaining path.  since it is not cleanly laid down it can be pulled off the bed when the nozzle later passes over it.

**Comment by m-vo** [comment]
... and if there are multiple brims (outside of the object and inside bigger holes) the first brim to print should be the outermost, so that the first string is catched. Therefore it IMHO also should be printed from 'away from the object' to 'close to the object' - finishing with a good line after the filament flow has stabilized.
